tests/c_api_tests/test_c_api_meta_data.cpp

```diff
@@ -17,24 +17,24 @@ TEST_CASE("C API Meta Data") {
     // get handle
     HandlePtr const unique_handle{PGM_create_handle()};
     PGM_Handle* hl = unique_handle.get();
-    auto const& meta = meta_data();
+    auto const& meta = meta_data;
```

Comment:
why not just make it `meta_data` everywhere?

Reply:
I was lazy to change.